### ISSUE TITLE
question about decouple loss

### ISSUE THREAD

**Issue by YvZheng** [thuml/predrnn-pytorch#11]
when I print decouple loss at core/train.py， I found the requires_grad of decouple loss is False！Does this mean that decouple loss does not play any role in backpropagation？

**Comment by wuhaixu2016** [comment]
Sorry, this is a typo when I reorganize the code. 

This can be fixed by only changing one line code:
change 'predrnn_memory_decoupling.py' line 113 to 'decouple_loss = torch.mean(torch.stack(decouple_loss, dim=0))'

This silly typo doesn't appear in our experiments. Thanks for your reminding～
